src/raftstore/store/store.rs

```diff
@@ -171,6 +179,12 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             Ok(true)
         }));
 
+        info!("[store {}] starts with {} regions, including {} tombstone and {} applying ones",
```

Comment:
+        info!("[store {}] starts with {} regions, including {} tombstones and {} applying regions",